service/build.gradle

```diff
@@ -125,9 +126,16 @@ sonarqube {
     property "sonar.projectName", "FPL :: Service"
     property "sonar.jacoco.reportPath", file("${project.buildDir}/jacoco/test.exec")
     property "sonar.jacoco.itReportPath", file("${project.buildDir}/jacoco/integration.exec")
+    property "sonar.host.url", "https://sonar.reform.hmcts.net/"
```

Comment:
Remove, we use sonar cloud now.

Reply:
Though we are use sonar cloud we use https://sonar.reform.hmcts.net/ for sonar.host.url which in turn cloud-native platform team pointing to sonar cloud https://sonarcloud.io

Reply:
Remove `sonar.host.url` property.